.github/workflows/test.yaml

```
@@ -0,0 +1,22 @@
+name: Docker
+
+on:
+  pull_request:
+    branches:
+      - main
```

```suggestion
on:
  push:
    branches: [main]
  pull_request:
    branches: [main]
```

Shouldn't we run tests on commit/push as well?